imblearn/under_sampling/__init__.py

```diff
@@ -3,17 +3,20 @@
 a dataset.
 """
 
-from .random_under_sampler import RandomUnderSampler
-from .tomek_links import TomekLinks
-from .cluster_centroids import ClusterCentroids
-from .nearmiss import NearMiss
-from .condensed_nearest_neighbour import CondensedNearestNeighbour
-from .one_sided_selection import OneSidedSelection
-from .neighbourhood_cleaning_rule import NeighbourhoodCleaningRule
-from .edited_nearest_neighbours import EditedNearestNeighbours
-from .edited_nearest_neighbours import RepeatedEditedNearestNeighbours
-from .edited_nearest_neighbours import AllKNN
-from .instance_hardness_threshold import InstanceHardnessThreshold
+from . import prototype_generation
+from .prototype_generation import ClusterCentroids
+
+from . import prototype_selection
```

Comment:
Yep, it is bringing the file from the each level undernith to under_sampling level